apps/odyssey-stats/src/widget/modules.jsx

```diff
@@ -2,41 +2,118 @@ import { ShortenedNumber } from '@automattic/components';
 import { protect, akismet } from '@automattic/components/src/icons';
 import classNames from 'classnames';
 import { useTranslate } from 'i18n-calypso';
+import { useState } from 'react';
+import wpcom from 'calypso/lib/wp';
 import useModuleDataQuery from '../hooks/use-module-data-query';
+import canCurrentUser from '../lib/can-current-user';
 
 import './modules.scss';
 
-function ModuleCard( { icon, title, value, isLoading, className = null } ) {
+function ModuleCard( {
+	icon,
+	title,
+	value,
+	activateProduct,
+	active,
+	className = null,
+	info = null,
+} ) {
+	const translate = useTranslate();
+	const [ disabled, setDisabled ] = useState( false );
 	return (
 		<div className={ classNames( 'stats-widget-module stats-widget-card', className ) }>
 			<div className="stats-widget-module__icon">{ icon }</div>
 			<div className="stats-widget-module__title">{ title }</div>
-			<div className="stats-widget-module__value">
-				{ isLoading ? '-' : <ShortenedNumber value={ value } /> }
-			</div>
+			{ ( active || ! activateProduct ) && (
+				<div className="stats-widget-module__value">
+					<ShortenedNumber value={ value } />
+				</div>
+			) }
+			{ ! active && activateProduct && (
```

Comment:
Could we resume the `isLoading` status for hiding the button or info text before the request is completed?
```suggestion
			{ ! isLoading && ! active && activateProduct && (
```

Reply:
Good point. Addressed in 50039f7352dbff3746f8d51d37810dcd72a59ee2